server/views/pages/wip/questions/questionPage.njk

```diff
@@ -57,9 +51,9 @@
         {% endif %}
 
         {% block fields %}
-          {% for key, fieldsOptions in fields %}
-            {% if fieldsOptions.component and not fieldsOptions.skip %}
-              {{ callAsMacro(fieldsOptions.component)(fieldsOptions) }}
+          {% for _fieldId, field in options.fields %}
+            {% if field.component and not field.skip %}
```

Comment:
`field.skip` is not a param that we have defined… i don't think it will do anything

Reply:
I'll remove that bit then 👌

Reply:
https://github.com/ministryofjustice/hmpps-incident-reporting/pull/223/commits/d2571e5fd83c3ee54260d014147a1950fb88a208